tools/data-handler/src/macros/index.ts

```diff
@@ -167,57 +172,36 @@ export async function evaluateMacros(
       result = compiled({});
 
       for (const macro of macroInstances) {
-        result = await macro.applyMacroResults(result);
+        result = await macro.applyMacroResults(result, context);
       }
       if (macroCount(result) === 0) {
         break;
       }
     } catch (err) {
-      return handleMacroError(err, emptyMacro);
+      return handleMacroError(err, emptyMacro, context);
```

Comment:
Could probably leave a comment since handleMacroError will throw on validate and otherwise return asciidoc

Reply:
Added